website/docs/utilities/dialog-primitive/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,33 @@
+## Conformance rating
```

Comment:
[blocker] This whole section needs to be reviewed: it refers to the `Modal` (or `Flyout`) but this is **not** the behaviour of the `DialogPrimitive`, it's more agnostic (there's practically no logic attached to it)

Reply:
Updated